mobile/src/reducers/organization/organization.reducer.ts

```diff
@@ -15,7 +16,8 @@ export const organizationEpics = combineEpics(
     loadDepartmentsEpic$ as any,
     loadChiefsEpic$ as any,
     loadDepartmentsFinishedEpic$ as any,
-    loadEmployeesForDepartmentEpic$ as any);
+    loadEmployeesForDepartmentEpic$ as any,
+    errorLoadFailedEpic$ as any);
```

Comment:
Also, move epic to the `errors\errors.reducer.ts` and add to the root epic:
```
export const errorsEpics = combineEpics(errorLoadFailedEpic$)
```
app.reducer.ts:
```
const rootEpic = combineEpics( helpdeskEpics as any, organizationEpics as any, errorsEpics);
```